gulpfile.js

```diff
@@ -246,6 +264,10 @@ gulp.task('build', [
   'build/zone.js',
   'build/zone.js.d.ts',
   'build/zone.min.js',
+  'build/zone-testing.js',
```

Comment:
the scripts needed are different depending if you are testing in node or in a browser.

For node, you need to start with zone-node.js, like Misko did in this PR:
https://github.com/angular/angular/pull/21053/files#diff-8ac936419da1e3d027833281525dba0e

Maybe we should also publish `zone-node-testing.js`?

Reply:
got it, I will make `zone-node-testing-bundle`